src/devices/CharacterLcd/LcdInterface.I2c.cs

```diff
@@ -59,7 +59,17 @@ public I2c(I2cDevice device)
 
             public override bool EightBitMode => true;
 
-            public override bool BacklightOn { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+            public override bool BacklightOn 
+            {
+                get
+                {
+                    // Not implemented / Not supported, but don't throw exceptions here
+                    return true;
+                }
+                set
+                {
+                }
```

Comment:
I'd leave this as it was instead override this in the new LCD and make getter return true/false and setter throw `NotSupportedException`

Reply:
That's a bit problematic, because then the client(s) always have to know which underlying interface they have. If you want to initialize the display (which usually means enabling the backlight), you don't want to test this at a higher level, I think.

Reply:
I don't feel strongly about it, fine to leave as is. We can re-iterate on that if we find someone with opposite opinion